package.json

```diff
@@ -333,6 +333,29 @@
                         "type": "string",
                         "default": "",
                         "description": "Defines which files in the workspace to ignore for Zip deploy. This applies to Zip deploy only, has no effect on other deployment methods."
+                    },
+                    "azureFunctions.projectRuntime": {
+                        "type": "string",
+                        "enum": [
```

Comment:
Why not allow "latest"?

Reply:
I chose to be consistent with the portal, which only allows '~1' or 'beta':
![screen shot 2017-12-14 at 11 09 03 am](https://user-images.githubusercontent.com/11282622/34009849-5388d740-e0bf-11e7-8f28-fc0c103f2956.png)

Let me know if you still think I should allow latest and I will address it separately